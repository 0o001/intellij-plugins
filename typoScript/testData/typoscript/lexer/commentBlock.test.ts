/*
commented = assgnment
commented < text
also = commented*/

/*        commented = assgnment
          commented < text
          also = commented*/