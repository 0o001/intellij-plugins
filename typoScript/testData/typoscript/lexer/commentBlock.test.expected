C_STYLE_COMMENT ('/*
commented = assgnment
commented < text
also = commented*/

/*        commented = assgnment
          commented < text
          also = commented*/')